src/auth/digest/Config.h

```diff
@@ -11,48 +11,13 @@
 
 #if HAVE_AUTH_MODULE_DIGEST
 
+#include "auth/digest/Nonce.h"
```

Comment:
If possible, please use a forward declaration of digest_nonce_h (and any other types) instead -- the header code below may not need to know digest_nonce_h class details... For example:

```suggestion
#include "auth/digest/forward.h"
```

Addressing this change request probably requires moving (temporary) digest_nonce_h declaration hack to forward.h as discussed in another change request: If that move is not feasible, then we cannot use forward.h here.